tests/robustness/failpoint/network.go

```diff
@@ -59,7 +59,7 @@ func (tb triggerBlackhole) Trigger(ctx context.Context, t *testing.T, member e2e
 func (tb triggerBlackhole) Available(config e2e.EtcdProcessClusterConfig, process e2e.EtcdProcess) bool {
 	// Avoid triggering failpoint if waiting for failpoint would take too long to fit into timeout.
 	// Number of required entries for snapshot depends on etcd configuration.
-	if tb.waitTillSnapshot && entriesToGuaranteeSnapshot(config) > 200 {
+	if tb.waitTillSnapshot && (entriesToGuaranteeSnapshot(config) > 200 || !e2e.CouldSetSnapshotCatchupEntries(process.Config().ExecPath)) {
```

Comment:
Don't like why this is needed. With your change allowing random member versions, snapshot-catchup-entries is configured on cluster, but not nesseserly every member. Only on members that version supports it. We test this edge case, however I don't like the fact it allows `snapshot-catchup-entries` to be now silently ignored.

Still, it's ok for now as I don't know a nice solution for this.

Reply:
Proposed to add a validation.